css/style.css

```diff
@@ -75,7 +76,282 @@ body {
   margin: 24px 0;
 }
 
+ul li {
+  text-align: center;
+  margin: 24px;
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 22px;
+  color: #fff;
+}
+
 .social-icon ul li {
   margin: 10px;
   padding: 10px;
 }
+
+/* work section */
+.works-section {
+  display: grid;
+  grid-template-columns: auto;
+  grid-template-rows: auto auto auto auto;
+  background-color: #fff;
+}
+
+.recentwork {
+  border-bottom: 1px solid #212121;
+  margin: 100px 24px;
+  max-width: 327px;
+}
+
+.recentwork p {
+  font-size: 40px;
+  font-weight: 400;
+  text-align: center;
+  font-family: 'Crete Round', serif;
+  margin-bottom: 50px;
+}
+
+.workone h2 {
+  font-size: 32px;
+  font-weight: 400;
+  font-family: 'Crete Round', serif;
+  margin: 10px 10px 10px 0;
+}
+
+.workone p {
+  font-size: 16px;
+  font-weight: 400;
+  font-family: 'Inter', sans-serif;
+  text-align: justify;
+}
+
+.workone .stack {
+  display: flex;
+  flex-direction: row;
+}
+
+.workone button {
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: 17px;
+  text-align: center;
+  width: 124px;
+  height: 48px;
+  color: #fff;
+  background-color: #ff6b00;
+  padding: 12px;
+  border: none;
+  margin-top: 5px;
+  cursor: pointer;
+}
+
+.worktwo-container {
+  display: grid;
+  grid-template-columns: auto;
+  grid-template-rows: auto auto auto auto auto auto;
+  margin-bottom: 30px;
+}
+
+.worktwo h2 {
+  padding: 84px 16px 16px;
+  font-family: 'Crete Round', sans-serif;
+  font-weight: 400;
+  font-size: 32px;
+  color: #fff;
+}
```

Comment:
- [ ] Here, According to the project requirement, every work section should be styled using a `display: flex` property. Please make sure to use `flexbox` to style all your work sections. This is to provide confidence that you can manipulate both flexbox and grid arrangements.